root/admin/wikidoc/EditWikiDoc.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * @flow
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *     |   i))  c   *\
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import FormRowText from '../../components/FormRowText';
+import FormSubmit from '../../components/FormSubmit';
+import Layout from '../../layout';
+import {withCatalystContext} from '../../context';
+
+type Props = {
+  +$c: CatalystContextT,
+  +current_version: number,
+  +form: FormT<{
+    +version: FieldT<string>,
+  }>,
+  +page: string,
+};
+
+const EditWikiDoc = withCatalystContext(({
+  $c,
+  current_version,
+  form,
+  page,
+}: Props) => (
+  <Layout fullWidth title={l('Update Page')}>
+    <div id="content">
+      <h1>{l('Update Page')}</h1>
+      <form action={$c.req.uri} method="post">
```

Comment:
`<FormCsrfToken />`

Reply:
Sure, I rebased on master to solve these conflicts, see https://github.com/metabrainz/musicbrainz-server/pull/1461/commits/cf44b4ebe17969f272de4fd19d05734cb9df1679#diff-4e9654eca8590dc78347cf2354e3b5c6R37